Docs/Generali/glossario/01_content.tex

```diff
@@ -230,6 +230,10 @@ \section{P}
     Nella grafica 3D, movimento della telecamera o della vista che consente 
     di spostarsi orizzontalmente o verticalmente in una scena senza modificarne 
     la prospettiva o l'orientamento.
+    \subsection{PB}
+    \label{PB}
+    (s. p. f.)\\
+    Acronimo di "Product Baseline", indica la seconda e ultima revisione di avanzamento del progetto effettuata dai docenti (committenti), durante la quale si verifica gli obiettivi funzionali e qualitativi del prodotto. Nel mezzo di questa fase, il prodotto viene presentato al proponente per l'accettazione come Minimum Viable Product (MVP).
```

Comment:
(committenti) durante (togli la virgola)
si verificano gli obiettivi 
Nel mezzo di questa fase -> sostituisci con "In questa fase il prodotto..."

Reply:
ho riaggiornato la definizione comunque